### DISCUSSION TITLE
Return CategoryPriceRange

### DISCUSSION THREAD

**Discussion by maleksandr** [scandipwa/scandipwa#2140] [General]
I guess Scandi should return back CategoryPriceRange module. Because Range slider is a must-have feature for ecommerce

**Comment by yeegor** [comment]
I recall it being removed for performance reasons. @alfredsgenkins knows more though.

**Comment by lianastaskevica** [comment]
It was removed when we migrated to the default Magento price attribute. It might become available as an extension soon:)
